### PR TITLE
[#XGB-163] Abstract and Mgmt Summary Review

### PR DESCRIPTION
**Pull request by raphiz**
I rephrased parts of the abstract, trying to realise the issues from our last meeting...

### CODE REVIEW

**Comment by raphiz** [comment]
aah, not yet done with it..

**Comment by fabianhauser** [comment]
@raphiz is it ready for review now, as the moved issue suggests? 😉

**Comment by raphiz** [comment]
ah, now it's ready!

File: documents/final-submission-document/final-submission-document.tex [hunk 1]
```diff
@@ -102,19 +102,24 @@
...
-The goal of this thesis is to engineer a specialised and platform independent XMPP-Grid Broker that configures existing XMPP Controllers (XMPP server). The XMPP-Grid Broker should enable administrators to configure XMPP-Grids in a usable and productive way.
-
-The focus of the application should be set on portability, extensibility and the security aspects in a production environment.\\
+The goal of this thesis is to design and implement an XMPP-Grid Broker to configure existing Controllers (XMPP server), focussing on portability, extensibility and the security aspects in a production environment.
+The  Broker application should enable administrators to configure XMPP-Grids in a usable and productive way.\\
 \vspace*{1ex}
 
 %Result
-We propose the architecture of an XMPP-Grid Broker to configure XMPP Controllers and present an implementation based on the Angular5 framework.
+Our proposed architecture earmarks a purely client-side application that communicates typically through a regular reverse proxy with the XMPP Controller over WebSockets or HTTP streams (BOSH).
```
Comment (fabianhauser) [inline documents/final-submission-document/final-submission-document.tex]: I find this sentence a bit long & hard to read.
Also, I'm not shure about the use of "earmarks". I assume you are? 😉

What do you think about splitting it up in two sentences like:

> Our proposed architecture earmarks a purely client-side application that communicates with the Controller over WebSockets or HTTP streams (BOSH).
> The Controller is typically protected by a reverse proxy, which also hosts our application.

File: documents/final-submission-document/final-submission-document.tex [hunk 2]
```diff
@@ -102,19 +102,24 @@
...
+Authentication is performed using mutual TLS authentication (client certificates) to conform to the IETF standard draft.
```
Comment (fabianhauser) [inline documents/final-submission-document/final-submission-document.tex]: I would remove client certificates here, as "mutual" also means verifying the server certificate (not only client certs).

File: documents/final-submission-document/final-submission-document.tex [hunk 3]
```diff
@@ -102,19 +102,24 @@
...
+The resulting application is implemented in TypeScript using the Angular5 framework.\\
+\vspace*{1ex}
 
-The resulting implementation enables administrators to create and configure communication Topics, apprehend the underlying hierarchy and manage permissions. Additionally, persistent items of Topics can be inspected and published.\\
+The resulting implementation enables administrators to create and configure communication Topics, apprehend the underlying hierarchy and manage permissions.
+Additionally, persistent items of Topics can be inspected and published.\\
```
Comment (fabianhauser) [inline documents/final-submission-document/final-submission-document.tex]: I just asked myself if we should write `Persistend Items` with capital letters here as "concept"?

File: documents/final-submission-document/final-submission-document.tex [hunk 4]
```diff
@@ -102,19 +102,24 @@
...
 \vspace*{1ex}
 
 % Consequences
-The XMPP-Grid Broker implementation incorporates the specified functionality, demonstrating that the architecture is robust. Some supplementary helpers, such as autocomplete or filtering, could not be implemented efficiently due to limitations of the underlying XMPP standards.
+The XMPP-Grid Broker implementation incorporates the specified functionality, resulting in a robust, ready-to-use solution.
+A few supplementary helpers, such as autocomplete or filtering, could not be implemented due to limitations of the underlying XMPP standards.
+These improvements can be realised in the future by enhancing the related XMPP standards or by specifying and implementing proprietary protocols
```
Comment (fabianhauser) [inline documents/final-submission-document/final-submission-document.tex]: What about rephrasing this sentence to:

> In the future, these improvements might be realised by enhancing [...]

I think it's easier to read and also makes clear, that we did not do that.

File: documents/final-submission-document/final-submission-document.tex (hunk 2 above)
Comment (raphiz) [reply documents/final-submission-document/final-submission-document.tex]: Yeah, sure, I only added it as a "denkstütze" to improve the readability

File: documents/final-submission-document/final-submission-document.tex (hunk 4 above)
Comment (raphiz) [reply documents/final-submission-document/final-submission-document.tex]: Sure

**Comment by raphiz** [comment]
you can merge if you agree with the latest additions

**Pull request merged**